src/core/installAddon.js

```diff
@@ -377,8 +382,15 @@ export class WithInstallHelpers extends React.Component {
     return _addonManager
       .getAddon(guid)
       .then(
-        (addon) => {
-          const status = addon.isActive && addon.isEnabled ? ENABLED : DISABLED;
+        ({ isActive, isEnabled }) => {
```

Comment:
It would be really nice to define types and get Flow coverage for `addonManager()` because it's undocumented and hard to understand.

Reply:
Filed https://github.com/mozilla/addons-frontend/issues/6088